src/s2wasm.h

```diff
@@ -372,8 +372,22 @@ class S2WasmBuilder {
 
   void prepare() {
     assert(nextStatic == globalBase); // we are the first allocation
+    // Allocate space for a user stack
+    staticAddresses[".stack"] = nextStatic;
```

Comment:
it would actually be more efficient to put the stack at the very end. The reason is that we will have baked constants to globals, and if they are first then they might be smaller than if they are after the stack, which might be a few MB. It's very possible that the stack being first would lead to extra bytes in the LEB128 encoding of those baked constants.